manifest-staging.yml

```
@@ -1,6 +1,7 @@
 ---
 defaults: &defaults
   buildpacks:
+    - https://github.com/andy-paine/env-map-buildpack.git
```

```suggestion
    - https://github.com/andy-paine/env-map-buildpack.git#ebb19052c377fa1ff1b2af5a390572237567a894
```
Prevents Andy from doing naughty things (unless he breaks SHA1)

---

I tried this but it failed to stage :( looks like cloud foundry doesn't support pinning to a sha1?

As an alternative we could fork his repo into alphagov but 🤢